packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -365,7 +365,8 @@ class CupertinoTextField extends StatefulWidget {
 
   /// The color to use when painting the cursor.
   ///
-  /// Defaults to the standard iOS blue color. Cannot be null.
+  /// Defaults to activeBlue for light theme, activeOrange for dark theme, or
+  /// settable primaryColor for customization.
```

Comment:
I would say `Defaults to the [CupertinoThemeData.primaryColor] of the ambient theme, which itself defaults to [CupertinoColors.activeBlue] in the light theme and [CupertinoColors.activeOrange] in the dark theme.`